mon-pix/app/templates/components/warning-banner.hbs

```diff
@@ -1,8 +1,18 @@
-{{#if isEnabled}}
+{{#if this.displayBanner}}
   <div class="warning-banner">
-    <a class="warning-banner__text" href="https://app.pix.fr">
-      {{fa-icon 'exclamation-triangle'}}
-      Cette plateforme est destinée aux tests internes à Pix. Vos pix ne sont pas comptabilisés.<br><span class="warning-banner__text--underline">Allez sur le site Pix.</span>
-    </a>
+    {{#if this.isStagingBannerEnabled}}
+      <a class="warning-banner__staging-text" href="https://app.pix.fr">
+        <FaIcon @icon="exclamation-triangle"></FaIcon>
+        Cette plateforme est destinée aux tests internes à Pix. Vos pix ne sont pas comptabilisés.
+        <br><span class="warning-banner__staging-text--underline">Allez sur le site Pix.</span>
+      </a>
+    {{/if}}
+    {{#if this.isProdProblemBannerEnabled}}
+      <div class="warning-banner__prod-problem-text">
+        <FaIcon @icon="exclamation-triangle"></FaIcon>
+        Nous rencontrons actuellement des ralentissements sur la plateforme pouvant affecter certains utilisateurs, <br/>
+        mais pas de panix, nos équipes sont sur le coup !
+      </div>
+    {{/if}}
```

Comment:
Avec `isProdProblemBannerEnabled`, j'ai l'impression que l'on mélange deux intentions : 
- connaître l'environnement courant
- connaître le message à afficher (non personnalisable pour le moment) 

Je verrais quelque chose comme : 

```suggestion
      <a class="warning-banner__text" href="https://app.pix.fr">
        <FaIcon @icon="exclamation-triangle"></FaIcon>
        {{#if this.isStaging}} 
	        Cette plateforme est destinée aux tests internes à Pix. Vos pix ne sont pas comptabilisés.
	        <br><span class="warning-banner__text--underline">Allez sur le site Pix.</span>
        {{#elseif this.isProduction}}
	        Nous rencontrons actuellement des ralentissements sur la plateforme pouvant affecter certains utilisateurs, <br/>
	        mais pas de panix, nos équipes sont sur le coup ! 
        {{/if}}
      </a>
```

Avec le retour de l'ancienne variable d'environnement `IS_WARNING_BANNER_ENABLED` qui permet de faire un lien direct avec le composant éponyme et deux méthodes voire une seule méthode `this.isEnvironment === "foo"` qui reflète l'environnement courant

Reply:
Ou alors on remplace tout ça par une variable d'environnement `WARNING_BANNER_CONTENT` et l'application n'a plus besoin de savoir s'il s'agit d'un problème de production, d'environnement de test, etc.

Reply:
Juste `WARNING_BANNER_CONTENT` ne suffirait pas car dans le cas de staging, on veut en plus pouvoir spécifier l'url de prod et y faire une redirection lorsque l'on clique sur la bannière. Par contre si on rend plus tard le texte configurable depuis Pix Admin, peut être y envisager une saisie html.

Reply:
> Juste `WARNING_BANNER_CONTENT` ne suffirait pas car dans le cas de staging, on veut en plus pouvoir spécifier l'url de prod et y faire une redirection lorsque l'on clique sur la bannière.
Je ne comprends pas en quoi c'est contradictoire avec `WARNING_BANNER_CONTENT` ?

Reply:
`WARNING_BANNER_CONTENT` contient bien le texte à afficher dans la bannière?
```hbs
{{#if this.warningBannerContent}}
  <div class="warning-banner__prod-problem-text">
    <FaIcon @icon="exclamation-triangle"></FaIcon>
    {{this.warningBannerContent}}
  </div>
{{/if}}
```
Mais sur staging, il y a aussi le lien vers la prod.

Reply:
Par exemple :

```
WARNING_BANNER_CONTENT="Cette plateforme est destinée aux tests internes à Pix. Vos pix ne sont pas comptabilisés.<br><a href="https://app.pix.fr">Allez sur le site Pix.</a>"
```